docs/en/README.md

```diff
@@ -0,0 +1,205 @@
+# Getting started
+
+Yii Db Migration is package for YiiFramework 3.0, which allows you to use migrations in your project, using Yii Db.
```

Comment:
I think most of it should be in main README and not to be moved away since that's the first things people read. In the new layout it's not really explicit about what package does from looking at the main README.

Reply:
fixed.